### PR TITLE
[OT171-21]Sign Up view design

### PR DESCRIPTION
**Pull request by frannasich**
### **Sign up view design added to the project**

- Added SignUpViewController with its xib
- Logo added to the view
- Placeholder added to the Mails label
- View labels conected with Controller

Captura de Pantalla 2022-03-18 a la(s) 14 09 27

image

### **Jira Ticket**
OT171-21

### CODE REVIEW

**Comment by frannasich** [comment]
I had to re-create another branch and make a pull request... is there a problem with deleting OT171-21_sign_up_view?

**Comment by gapoj** [comment]
missing the name field on the form

**Comment by gapoj** [comment]
DS_Store should not be on the repo

**Comment by frannasich** [comment]
Ds_Store was pulled from the main

**Comment by frannasich** [comment]
Sign up view is update

File: SomosMasApp/SomosMasApp/Info.plist [hunk 1]
```diff
@@ -15,8 +15,6 @@
 					<string>Default Configuration</string>
 					<key>UISceneDelegateClassName</key>
 					<string>$(PRODUCT_MODULE_NAME).SceneDelegate</string>
-					<key>UISceneStoryboardFile</key>
```
Comment (gapoj) [inline SomosMasApp/SomosMasApp/Info.plist:18]: why?

**Review by gapoj — CHANGES_REQUESTED** [review]
You should made only the requested changes, you should not affect the navigation

**Pull request merged**